<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestVDraw</class>
 <widget class="QDialog" name="TestVDraw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="circleTestButton">
       <property name="text">
        <string>Circles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lineTestButton">
       <property name="text">
        <string>Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rectangleTestButton">
       <property name="text">
        <string>Rectangles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ellipsesTestButto">
       <property name="text">
        <string>Ellipses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>LIne width</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSpinBox" name="circleCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="linesCoun">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rectanglesCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ellipsesCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lineWidth">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
